be/src/olap/memtable.cpp

```diff
@@ -101,19 +101,14 @@ void MemTable::_init_agg_functions(const vectorized::Block* block) {
     }
 
     for (uint32_t cid = _schema->num_key_columns(); cid < _schema->num_columns(); ++cid) {
+        size_t alignment_of_state = _agg_functions[cid]->align_of_data();
+        size_t module = _total_size_of_aggregate_states % alignment_of_state;
+        if (module) {
+            _total_size_of_aggregate_states += alignment_of_state - module;
+        }
```

Comment:
DCHECK(alignment_of_state & (alignment_of_state -1)  == 0);
_total_size_of_aggregate_states = (_total_size_of_aggregate_states + alignment_of_state -1 )  & (~(alignment_of_state -1));

Reply:
alignment_of_state just is `sizeof(FunctionData)`, not necessarily power of 2